pop-api/Cargo.toml

```diff
@@ -40,3 +40,4 @@ std = [
 	"pop-primitives/std",
 	"sp-io/std",
 ]
+messaging = [	]
```

Comment:
unfortunately, zepter or taplo will yell you at for an empty feature. Populating with some default feature works to resolve it.